sphinx_scality/static/css/theme.css

```diff
@@ -3,3 +3,122 @@
 
 @import url("./fonts.css");
 @import url("./current.css");
+
+
+/****** header******/
+
+div.header {
+    margin: 0 8% 0 7%;
+}
+
+
+
+
+/**** Header logo arrow search *****/
+
+img.baselogo {
+    width: 125px;
+}
+div.baselogo > a {
+  
```

Comment:
Let's clean spacing please: each CSS rule should be surrounded with empty lines (1 above, 1 below), and a rule shouldn't contain empty lines.